src/bridge_lcd.cpp

```diff
@@ -444,7 +436,7 @@ void bridge_lcd::print_line(const char *left_text, const char *middle_text, cons
     yield();
     tft->drawString(middle_text, 134, starting_pixel_row, GFXFF);
     yield();
-    tft->drawString(right_text, 320 - tft->textWidth(right_text, GFXFF), starting_pixel_row, GFXFF);
+    tft->drawString(right_text, 320-tft->textWidth(right_text,GFXFF), starting_pixel_row, GFXFF);
```

Comment:
This format change is not in line with the rest of the program, similar to the others.